docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/05_instrument_quality_assurance.md

```diff
@@ -0,0 +1,8 @@
+## Instrument Quality Assurance
+
+### Double Data Entry
+Double Data Entry (DDE) is the practice of requiring users to manually input the same instrument data twice, in different isolated locations, in order to reduce the risk of error in data entry by flagging differences between the primary and secondary data entry. For DDE-enabled instruments, a participant's instrument data collection at a given visit cannot be finalized in LORIS unless both the first data entry form and second (double) data entry form are completed.
```

Comment:
I don't think there's a location requirement for DDE..

Reply:
i just meant on different links, not locations as in sites. removed anyways